cx_Freeze/freezer.py

```diff
@@ -89,30 +103,10 @@ def __init__(
         self.zipIncludePackages = zipIncludePackages
         self.zipExcludePackages = zipExcludePackages
         self._VerifyConfiguration()
+        return
```

Comment:
this 'return' is unnecessary

Reply:
I agree, technically. I like to put returns at the end of all functions, just for visual clarity.  Do you feel strongly?

Reply:
I did not understand the question. I do not perceive visual clarity. I've been following the pep8 and other rules and got used to them.

Reply:
I was just asking if you care a lot about the "return" at the function end.  I tend to have include the return because my IDE highlights it so I clearly see where the function ends, but it is not critical if you want it gone.

Reply:
I'll remove all of them.

Reply:
Fine with me.